core/primitives/src/trie_key.rs

```diff
@@ -33,6 +33,8 @@ pub(crate) mod col {
     pub const DELAYED_RECEIPT_INDICES: &[u8] = &[7];
     /// This column id is used when storing delayed receipts, because the shard is overwhelmed.
     pub const DELAYED_RECEIPT: &[u8] = &[8];
+    /// This column id is used when storing Key-Value data from a contract on an `account_id`.
+    pub const CONTRACT_DATA: &[u8] = &[9];
```

Comment:
It's not required, but it's relevant.

It doesn't require a hard fork. Since we start from genesis, the trie is always empty at the beginning.

This change just made it easier to separate account values from contract data.

Reply:
"Since we start from genesis" - I call this hard fork, am I using the wrong terminology? This change whenever it lands will require us to start from genesis.

Reply:
Any logic change requires hard fork, even if we don't change genesis config. So it's true

Reply:
My argument is just that I recall a suggestion to somehow mark such PRs, so we can keep track of those

@ilblackdragon Did we get anywhere with this policy of marking PRs that require hard forks?

Reply:
This is not a hardfork, but it is invasive.
Hardfork is when the node with new version of the code cannot join network with the old version of the code, because it is going to be slashed.

This change basically prohibits a new node to run on the state produced by the old code, which means it needs a migration script.